frontend/src/app/main/data/workspace/groups.cljs

```diff
@@ -90,7 +90,10 @@
         group     (cts/setup-shape {:type :group
                                     :name gname
                                     :shapes (mapv :id shapes)
-                                    :selrect selrect
+                                    :x (:x selrect)
```

Comment:
You also can leave selrect here, so it will be reused instead of creating a new one

Reply:
done